plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py

```diff
@@ -642,6 +657,21 @@ def _export_gepknoten(self):
 
     def _export_infrastrukturknoten(self):
         query = self.tww_session.query(self.model_classes_tww_ag6496.gepknoten)
+        if self.filtered:
+            query = query.join(
+                self.model_classes_tww_ag6496.gephaltung,
+                or_(
+                    self.model_classes_tww_ag6496.gepknoten.obj_id
+                    == self.model_classes_tww_ag6496.gephaltung.startknoten,
+                    self.model_classes_tww_ag6496.gepknoten.obj_id
+                    == self.model_classes_tww_ag6496.gephaltung.endknoten,
+                ),
+            ).filter(
+                or_(
+                    self.model_classes_tww_ag6496.gephaltung.obj_id.in_(self.subset_ids),
+                    self.model_classes_tww_ag6496.gepknoten.obj_id.in_(self.subset_ids),
+                )
+            )
```

Comment:
Q: Kann dieser Filter ausgelagert und wiederverwendet werden? z.B weiter oben in `_export_gepknoten`. Betrifft auch die weiteren Filter, die mehrmals gleich implementiert werden.